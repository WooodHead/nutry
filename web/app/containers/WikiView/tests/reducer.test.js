
import { fromJS } from 'immutable';
import wikiViewReducer from '../reducer';


import {
  defaultAction,
     } from '../actions';

describe('wikiViewReducer', () => {
  it('returns the initial state', () => {
    expect(wikiViewReducer(undefined, {})).toEqual(fromJS({}));
  });
  it('handles default action', () => {
    expect(wikiViewReducer(undefined, defaultAction())).toMatchSnapshot();
  });
});
