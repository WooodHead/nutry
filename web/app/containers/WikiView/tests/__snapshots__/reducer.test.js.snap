// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wikiViewReducer handles default action 1`] = `Immutable.Map {}`;
